### ISSUE TITLE
Add details to data in AuthenticationAuditListener.onAuthenticationFailureEvent

### ISSUE THREAD

**Issue by Kwaq** [spring-projects/spring-boot#4819]
Can Authentication details object be added to the data map sent in AUTHENTICATION_FAILURE? Same as in AUTHENTICATION_SUCCESS.

**Comment by philwebb** [comment]
Seems like a sensible addition. Have you [signed the CLA](https://github.com/spring-projects/spring-boot/blob/master/CONTRIBUTING.adoc#sign-the-contributor-license-agreement)?

**Comment by Kwaq** [comment]
Not until just now :) My confirmation number is 156420151223091252.

**Comment by wilkinsona** [comment]
@Kwaq I think @philwebb mistook this issue for a pull request. Unless I've missed it (entirely possible), I don't think we have a pull request that makes your proposed change. Would you like to submit one?

**Comment by Kwaq** [comment]
I've submitted pull request. Second commit fixes unwanted imports reorganization.

**Comment by wilkinsona** [comment]
Thanks, @Kwaq. Closing in favour of the PR.
